gallery/src/components/NxDrawer/NxDrawerDefaultOpenExample.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { NxDrawer,
+  NxBackButton,
+  NxButton,
+  NxH1,
+  NxP,
+  NxPageMain,
+  NxPageTitle,
+  useToggle
+} from '@sonatype/react-shared-components';
+
+export default function NxDrawerDefaultOpenExample() {
```

Comment:
I think a more useful example for this case would be one that still only opens the drawer when you push the button, but where it does so not by toggling `open` but by conditionally rendering the entire drawer.  And then on close it would toggle `open` and unmount the drawer after it's done closing.  That's the most typical pattern that consuming code will need to follow, so we should demonstrate it.

Reply:
> I think a more useful example for this case would be one that still only opens the drawer when you push the button, but where it does so not by toggling `open` but by conditionally rendering the entire drawer. And then on close it would toggle `open` and unmount the drawer after it's done closing. That's the most typical pattern that consuming code will need to follow, so we should demonstrate it.

Thanks, I've updated the example here: af9a78c1c93bc35faff9bb813a5494e6157ef03e